js/tiles.js

```diff
@@ -293,3 +293,11 @@ const door = {
     },
   }),
 };
+outdoor: Object.assign({}, OpenDoors, {
+   	canEnterFromTheRight() {return false;},
+      canLeaveToTheRight() {return false;},
+      createImages: function() {
+          this.wallRight = this.createImage("tiles/rooms/wall/outwall.svg");
+          this.ground = this.createImage("tiles/rooms/floor/outdoor.svg");
+      },
+}),
```

Comment:
Remove the `,`

Reply:
It's okay to leave that there. But the position is wrong :)